route_following_plugin/include/route_following_plugin.h

```diff
@@ -125,6 +125,7 @@ namespace route_following_plugin
         ros::Publisher plugin_discovery_pub_;
         ros::Subscriber pose_sub_;
         ros::Subscriber twist_sub_;
+        ros::Timer discovery_pub_timer_;
```

Comment:
See above, I assume this was copy pasted to save time

Reply:
This is a discovery pub timer though. If you look in the cpp file you can see it only publishes the plugin discovery message.